recipes/extra-cmake-modules/all/conanfile.py

```diff
@@ -0,0 +1,42 @@
+import os
+from conans import ConanFile, CMake, tools
+
+class ExtracmakemodulesConan(ConanFile):
+    name = "extra-cmake-modules"
```

Comment:
Should the name be namespaced?
e.g. `kde-extra-cmake-modules`

`extra-cmake-modules` sounds generic.

This is just a question, let's hear the opinion of the others.

Reply:
> extra-cmake-modules sounds generic.

Indeed, I didn't about this project, but when searching, it's well referenced to KDE, both Debian packages and Qt points to KDE page. Also, Conda offers extra-cmake-modules by the same name. So I think extra-cmake-modules is okay, as there is no other popular project using the same name or similar.

Reply:
KDE does prefix almost everything with a `k`. What a shame that they haven't done here the same and leave us with this unfortunate choice 😄 

The package got into many Linux distros with the unchanged name. I guess we can leave it at that

https://pkgs.org/download/extra-cmake-modules